src/BookingsRow.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useCallback, useEffect } from "react";
+import moment from "moment/moment";
+
+const BookingsRow = ({ guest, changeGuestProfile }) => {
+  const [isSelected, setIsSelected] = useState(false);
+  const [finalData, setFinalData] = useState(null);
+
+  useEffect(() => {
+    const fetchGuest = async () => {
+      const response = await fetch(
+        `https://cyf-react.glitch.me/customers/${guest.id}`
+      );
+      const data = await response.json();
+      setFinalData(data);
+    };
+    fetchGuest();
+  }, [guest.id]);
+
+  const calculateNights = useCallback((firstDate, secondDate) => {
+    let first = moment(firstDate, "YYYY-MM-DD");
+    let second = moment(secondDate, "YYYY-MM-DD");
```

Comment:
It is generally a better idea to use `const` where we can, if we are not going to be reassigning variables.